docs/pages/guides/schema-validation/syntax.mdx

```diff
@@ -49,8 +49,8 @@ A sample schema using only scalar types could look like this:
 type Storage {
   name: string
   age: number
-  height: number
```

Comment:
Numbers were already demoed by `age`, no need to also use `height`. Instead, I'm demoing the ability to use unions with the new `favoritePet` field.